youtube_dl/extractor/crunchyroll.py

```diff
@@ -5,6 +5,7 @@
 import json
 import base64
 import zlib
+import cfscrape
```

Comment:
Should handle ImportError and display proper messages to users. See https://github.com/rg3/youtube-dl/blob/master/youtube_dl/downloader/hls.py#L6 for an previous example.